### ISSUE TITLE
Some confusions about the loss in mle_scoring.py

### ISSUE THREAD

**Issue by markict123** [hkust-zhiyao/RTL-Coder#6]
I'm interested in the loss function put forward in the paper. When I read relevant codes, the implementation made me a little confused.
https://github.com/hkust-zhiyao/RTL-Coder/blob/3394cce416fb0d70f76d81f809be5d0c32de0c55/train/mle_scoring.py#L254
It seems that comp_loss and domain_loss should be added together without bracket according to the paper. 
Is there any mistake? Thank you for checking!

**Comment by DevinShang** [comment]
Ho markict123,! You are correct. If we follow the paper strictly, this bracket should not exist. 
But we think the code of line 254 has no conflict with the paper because we can always adjust the self.args.compare_weight to balance the two loss parts, i.e., comp_loss and domain_loss.
